### PR TITLE
chore(GlobalPage): restricted allowed background colors

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Changes `GlobalPage`'s `backgroundColor` prop to be only one of the ones we know we want.

I keep accidentally giving it `"blue-0"` instead of `"paleBlue"`...

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.98-alpha.d85bbb.0<br/>@codecademy/gamut-labs@24.1.2-alpha.d85bbb.0<br/>@codecademy/styleguide@47.1.2-alpha.d85bbb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6154e491dfbf2e21161a6067--gamut-preview.netlify.app>https://6154e491dfbf2e21161a6067--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6154e491dfbf2e21161a6067>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalPage/index.tsx [hunk 1]
```diff
@@ -3,14 +3,23 @@ import {
...
 import { GlobalHeader, GlobalHeaderProps } from '../GlobalHeader';
 
+export type GlobalPageBackgroundColor =
+  | 'beige'
+  | 'background'
+  | 'navy'
+  | 'paleBlue'
+  | 'paleGreen'
+  | 'paleYellow'
+  | 'white';
```
Comment (dreamwasp) [inline packages/gamut-labs/src/GlobalPage/index.tsx:19]: @songstephen, how does this list look to you?

Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/GlobalPage/index.tsx]: Talked with Stephen on Slack and got a 👍

**Comment by JoshuaKGoldberg** [comment]
@dreamwasp you got it! https://github.com/codecademy-engineering/Codecademy/pull/27806 -> https://pr-27806.dev-eks.codecademy.com

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.121-alpha.8afeae.0<br/>@codecademy/gamut-labs@25.2.3-alpha.8afeae.0<br/>@codecademy/styleguide@48.0.16-alpha.8afeae.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.7-alpha.578559.0<br/>@codecademy/gamut-labs@28.0.2-alpha.578559.0<br/>@codecademy/styleguide@50.1.4-alpha.578559.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6182be3fb6e9883593cd4d59--gamut-preview.netlify.app>https://6182be3fb6e9883593cd4d59--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6182be3fb6e9883593cd4d59>Deploy Logs</a></p>

**Pull request merged**